tests/testthat/_snaps/attr-nodes/yarn.md

```diff
@@ -0,0 +1,23 @@
+---
+title: basic curly
+---
+
+# preface {#pre-face .unnumbered}
+
+hello
+
+I like {xml2} but of course {tinkr} is even cooler!
+
+Images that use pandoc style will have curlies with content that should be translated and should be protected.
+
+![a pretty kitten](https://placekitten.com/200/300){#kitteh alt='a picture of a kitten'}
+
+![a pretty puppy](https://placedog.net/200/300){#dog alt="a picture
+of a dog"}
+
+
+\[a span with attributes\]{.span-with-attributes
```

Comment:
is this escaping expected?

Reply:
Unfortunately, yes: https://github.com/r-lib/commonmark/issues/20 

I'm going to try to implement this today. It should be a matter of backtracking and finding the square braces that were not escaped in the original document and then tagging them with 'asis' tags.